tests/services/components/test_access_component.py

```diff
@@ -34,11 +35,24 @@ def test_access_component_unknown_owner(
     record = RDMRecord.create(minimal_record)
     component = AccessComponent(RDMRecordService())
 
+    # both should work, since 'access.owned_by' is ignored for anybody
+    # other than system processes
+    component.create(identity_simple, minimal_record, record)
+    component.update_draft(identity_simple, minimal_record, record)
+
+
+def test_access_component_unknown_owner_with_system_process(
```

Comment:
split up the test in two variants: with a normal user's identity, and with the system identity... since with the new changes, these should behave differently